### DISCUSSION TITLE
Not getting the option to allow email authentication

### DISCUSSION THREAD

**Discussion by dignin** [outline/outline#4007] [Self Hosting]
I've enabled the Google API, and it says it doesn't work with private accounts. So I want to use the email a magic link function. I've configured the SMTP settings, but get no option to enable it.

I'm using the latest tagged version 0.65.2. on Ubuntu 20 with Apache2 Reverse Proxy.

**Comment by Interperle** [comment]
I'm having the same problem, did you manage to solve it?

**Reply by fabiansiegel** [reply]
For me, it only appeared after setting up another authentication. Did you solve it?

**Reply by ovizii** [reply]
I'm struggling with the same issue. It seems impossible to use email link authentication as the only authentication method out of the box :-(
